src/engine/QueryExecutionTree.cpp

```diff
@@ -210,7 +215,12 @@ nlohmann::json QueryExecutionTree::writeJsonTable(
   for (size_t i = from; i < upperBound; ++i) {
     json.emplace_back();
     auto& row = json.back();
-    for (const auto& idx : validIndices) {
+    for (const auto& opt : validIndices) {
+      if (!opt) {
+        row.emplace_back(optToJson(std::optional<string>{}));
```

Comment:
Could this be just an empty string?

Reply:
I guess since JSON allows Null and this one returns JSON I would expect a null, or am I missing something?